Makefile

```diff
@@ -41,7 +41,7 @@ $(BINARY):
 
 DOCKER_BUILD_PLATFORM         ?= linux/amd64,linux/arm64,linux/ppc64le,linux/arm/v6,linux/arm/v7
 DOCKER_BUILD_RUNTIME_IMAGE    ?= alpine:3.18
-DOCKER_BUILDX_ARGS            ?= --build-arg RUNTIME_IMAGE=${DOCKER_BUILD_RUNTIME_IMAGE}
+DOCKER_BUILDX_ARGS            ?= --build-arg RUNTIME_IMAGE=${DOCKER_BUILD_RUNTIME_IMAGE} --provenance=false
```

Comment:
What does this new arg do?

Reply:
It is used to control how much metadata is attached to images to figure out their origin:

https://docs.docker.com/build/attestations/
https://docs.docker.com/build/attestations/slsa-provenance/

I was trying to figure out why the automated build and the build from my machine produces these `unknown` archs for each regular arch that is build:
![image](https://github.com/oauth2-proxy/oauth2-proxy/assets/18513179/6f4f1bef-1e53-4181-8d5b-15909629cd23)

It seems like something is differently setup on your system as all previous builds to quay.io from your system do not contain any of these `unknown` archs.

Reply:
I don't believe it is an issue but something is different between how I build the images and how you build images on your system.

You find the some "issue" for all nightly builds as well:
https://quay.io/repository/oauth2-proxy/oauth2-proxy-nightly?tab=tags